theme/static/css/hydroshare_core.css

```diff
@@ -4560,7 +4560,17 @@ span.caption-file-type {
 
 
 .btn-remove {
-	font-size: 18px;
+	font-size: 14px;
```

Comment:
be careful altering this class -- it is used in many elements!
If you're going to modify it, I would do a search in the codebase for `btn-remove` to make sure that you're not breaking styles in other places